codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGeneratorTests.kt

```diff
@@ -9,7 +9,7 @@ class PresignerGeneratorTests {
     @Test
     fun `001 presignable on getFooInput`() {
         val context = setupTests("awsrestjson1/presignable.smithy", "smithy.swift.traits#Example")
-        val contents = TestUtils.getFileContents(context.manifest, "/Example/models/GetFooInput+Presigner.swift")
+        val contents = TestUtils.getFileContents(context.manifest, "Sources/Example/models/GetFooInput+Presigner.swift")
```

Comment:
Test files follow.  In general, tests are adjusted for new file locations and more precise symbols in generated code.